specs/users/users_spec.js

```diff
@@ -0,0 +1,60 @@
+describe('Edit Profile', function() {
+  var originalTimeout;
+  beforeEach(function() {
+    originalTimeout = jasmine.DEFAULT_TIMEOUT_INTERVAL;
+    jasmine.DEFAULT_TIMEOUT_INTERVAL = 200000;
+  })
+
+  it('should have a heading', function() {
+    browser.get('http://localhost:5000/#/login');
+    element(by.linkText('Sign in with Google')).click();
+    browser.getAllWindowHandles().then(function (handles) {
+      newWindowHandle = handles[1];
+      browser.driver.switchTo().window(newWindowHandle).then(function () {
+        browser.driver.findElement(by.id('Email')).sendKeys("framgia.fepas.test@gmail.com");
+        browser.sleep(2000);
+        browser.driver.findElement(by.id('next')).click();
+        browser.sleep(2000)
+        browser.driver.findElement(by.id('Passwd')).sendKeys("Fep@s2016");
+        browser.sleep(2000);
+        browser.driver.findElement(by.id('signIn')).click();
+        browser.sleep(2000);
+      });
+      backWindowHandle = handles[0];
+      browser.switchTo().window(backWindowHandle)
+    });
+    browser.sleep(3000);
+    browser.get('http://localhost:5000/#/users/edit');
+    var heading = element(by.css('h1'))
+
+    expect(heading.getText()).toEqual('Edit profile');
+  });
+
+  var name = 'name update';
+  var division = 'division';
+  var age = '24';
+
+  it('can update profile', function(){
+    browser.get('http://localhost:5000/#/users/edit');
+
+    element(by.model('user.info.name')).sendKeys(name);
+    element(by.model('user.info.division')).sendKeys(division);
+    element(by.model('user.info.age')).sendKeys(age);
+    element(by.buttonText('Save Changes')).click();
+    browser.sleep(2000);
+
+    expect(browser.getCurrentUrl()).toMatch('http://localhost:5000/#/');
+  });
+
+  it('check value change after update', function(){
```

Comment:
Tưởng phải gộp cái `it` này với cái `it` trên
với lại name, division, age phải random thì mới đảm bảo là update đúng chứ?
Nếu không chạy lại lần 2 thì update giá trị như cũ mà?